docs/sql/array.md

```diff
@@ -57,6 +57,14 @@ will produce a collection whose columns are the fields of the
 structure, instead of a collection with a single structure-typed
 column.
 
+## Comparison Operations on Arrays
+
+Comparison operations (`=`, `<>`, `!=`, `>`, `<`, `>=`, `<=`) in arrays occur **lexicographically**, from left to right. The elements are compared one by one until a difference is found. Arrays can be compared even if they are different in size.
+
+**Examples:**
+- `n_arr1 = [45] > n_arr2 = [22]`
```

Comment:
There are no variables in SQL, so just write [45] > [22].
Also, there is no None, use NULL.